pyproject.toml

```diff
@@ -5,7 +5,10 @@ description = "Bayesian Optimization package"
 authors = ["Fernando Nogueira"]
 license = "MIT"
 readme = "README.md"
-packages = [{include = "bayes_opt"}]
+packages = [
+    {include = "bayes_opt"},
+    {include = "bayes_opt/py.typed"},
```

Comment:
It seems this explicit include [might not actually be necessary](https://jugmac00.github.io/blog/bite-my-shiny-type-annotated-library/#poetry).

Reply:
In local, I ran the `poetry build` command to verify that `py.typed` was included.
But I prefer to be explicit rather than implicit.
Of course, this is just my personal preference, but if you prefer simplicity, it seems better to remove it.
I'll fix it as soon as you point it out.

Reply:
I fixed it